ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+### Prerequisites
+ 
+ * [ ] Did you check if your issue has already being reported?
+ * [ ] Are you running the latest version?
+ * [ ] Did you check the [Official Create React App User Guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md)?
```

Comment:
Awesome @BrianCortes. Hey,we are not using create react app on the server side. So, I believe you can remove this line. Besides that, I think we can start with it! Thanks ❤️

Reply:
ok , @jvcjunior no problem !